src/patterns/components/inputs/angular/global-info.hbs

```diff
@@ -0,0 +1,70 @@
+<p class="sprk-u-mbm">The input components expect html for labels and inputs to be inside the provided sprk elements. We
+do it this way so that you can use standard html attributes, such as disabled and readonly. Also, with the markup
+exposed, you're able to bind your Angular form logic directly.</p>
+
+<p class="sprk-u-mbm">The inputs are grouped into two groups, those that make use of sprk-input-container, and those
+that use sprk-selection-container. The sprk-input-container inputs expect the html for labels and inputs whereas the
+sprk-selection-containers expect there to be sprk-selection-item-containers inside. Sprk-selection-item-containers also
```

Comment:
I think @bobwise made a code style for code. Wondering if we should make use of it for the stuff we are doing?